docs/tsp.md

```diff
@@ -0,0 +1,167 @@
+# TSP Messages
+
+**Version 0.1**
+
+The TSP messages (Transit Signal Priority) are used to communicate with BYMs signal priority solution.
+
+The messages are encoded as json payloads, and you can validate any payload by using the corresponding JSON schema 
+definition found in the corresponding [JSON schema repository](#json-schemas).
+
+MQTT is used as the transport mechanism for all TSP messages. MQTT is a simple pub-sub based messaging protocol, that 
+works on top of TCP/IP as well as on websockets.
+
+## Summary
+The following is a summary of the messages
+
+| Local topic                        | Change     | Receiver    | Name                                     | Comments                                                                                                                                                                                                  |
+|------------------------------------|------------|-------------|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| bym/ruter/[vehicleRef]/vm          |-           | BYM         | [VehicleMonitoring](#VehicleMonitoring)  |                                                                                                                                                                                                           |
+| bym/ruter/[vehicleRef]/journey     |-           | BYM         | [Journey](#Journey)                      |                                                                                                                                                                                                           |
+| ruter/bym/[vehcileRef]/tspack      |-           | Ruter       | [TSP ack](#TSPack)                       |                                                                                                                                                                                                           |
+
+## Description of messages
+
+### VehicleMonitoring
+
+|               |                                                     |
+|---------------|-----------------------------------------------------|
+| Name          | VehicleMonitoring                                   |
```

Comment:
Denne raden bør være over linjen for å være tittlene på kolonnene.

Reply:
Jeg ønsker tabell uten tittel på kolonnene
Det finnes ikke i vanlig markdown. Jeg syns resultatet ved å ikke skrive noe ble helt _OK_